bin/psgi-bin/otobo.psgi

```diff
@@ -455,7 +469,7 @@ my $RedirectOtoboApp = sub {
     $Res->redirect($Redirect);
 
     # send the PSGI response arrayref
-    return $Res->finalize();
+    return $Res->finalize;
```

Comment:
Out of curiosity: Is there a technical reason or is this code styling? If it is for styling purposes, should I adapt this syntax?

Reply:
There is no special reason, just a matter of taste. I like it because it IMHO emphasizes that this a method call, but that is purely subjective.